appserver/monitoring-console/core/src/main/java/fish/payara/monitoring/store/ConsumingMonitoringDataCollector.java

```diff
@@ -46,31 +46,28 @@
 import fish.payara.monitoring.collect.MonitoringDataCollector;
 
 /**
- * A {@link MonitoringDataCollector} that is used as an adapter to {@link MonitoringDataSink} abstraction.
+ * A {@link MonitoringDataCollector} that is used as an adapter to {@link MonitoringDataConsumer} abstraction.
  *
  * @author Jan Bernitt
  */
-public class SinkDataCollector implements MonitoringDataCollector {
```

Comment:
NB: I never liked the naming - calling it a consumer is more understandable I think.